### PR TITLE
[Maintenance][Behat] Autoload calendar contexts into container

### PR DESCRIPTION
**Pull request by NoResponseMate**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13 |
| Bug fix?        | kinda                                                       |
| New feature?    | no                                                       |
| BC breaks?      | no                                                       |
| Deprecations?   | no |
| Related tickets | cause https://github.com/Sylius/Calendar/pull/13  |
| License         | MIT                                                          |

Long story short, it's a BC layer for `sylius/calendar:^0.5`.

It's either fixing it here, almost at the source, or in every app/plugin that uses Behat with our contexts.

### CODE REVIEW

**Review by Wojdylak — APPROVED** [review]

**Review by TheMilek — APPROVED** [review]

**Review by Rafikooo — APPROVED** [review]

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thank you, Jan! :1st_place_medal:
